tests/Resources/Common.robot

```diff
@@ -143,3 +144,16 @@ Clean Resource YAML Before Creating It
     Remove From Dictionary    ${clean_yaml_data}[metadata]  managedFields  resourceVersion  uid  creationTimestamp  annotations
     [Return]   ${clean_yaml_data}
 
+Wait Until OCM Cluster Page Is Loaded
```

Comment:
I think this keyword, `Login to OCM` and `Wait OCM Splash Page` are already implemented in tests/Resources/Page/HybridCloudConsole/HCCLogin.robot

Reply:
nope. they are different. The file you mentioned contains `Login to HCC` and `Wait For HCC Splash Page`

Reply:
OCM keyword moved to a dedicated resource file